api/src/pcapi/core/offers/api.py

```diff
@@ -2144,3 +2145,27 @@ def update_event_opening_hours(
                     eventOpeningHours=event, weekday=models.Weekday[weekday], timeSpans=time_spans
                 )
             )
+
+
+def delete_event_opening_hours(event: offers_models.EventOpeningHours) -> None:
```

Comment:
Détail: je propose de pas nommer event un truc qui n'est pas un event
```suggestion
def delete_event_opening_hours(event_opening_hours: offers_models.EventOpeningHours) -> None: